src/style.css

```diff
@@ -2,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+  font-weight: bold;
+}
+
+:root {
+  --font-family: Avenir, Arial, sans-serif;
+  --font-color: #3d4e61;
+  --background-color: #fff;
+  --border-color: #e1e7ed;
+  --box-shadow: rgba(61, 78, 97, 0.1);
+  --hover-box-shadow: rgba(61, 78, 97, 0.2);
+  --placeholder-color: #3d4e61;
+  --search-icon-size: 19px;
+  --search-icon-small-size: 11px;
+  --search-bar-height: 70px;
+  --search-bar-small-height: 42px;
+  --font-size: 16px;
+  --font-size-small: 14px;
+}
+
+body {
+  padding: 0;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  font-family: var(--font-family);
+  font-size: var(--font-size);
+  color: var(--font-color);
+  background-color: var(--background-color);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: var(--search-icon-size);
+  background-position: 25px;
+  width: 100%;
+  height: var(--search-bar-height);
+  padding-left: 62px;
```

Comment:
Avoid fixing the padding size. Instead, consider using relative units or a variable if the design allows for flexibility. This can make the design more responsive and easier to maintain.